src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -32,6 +32,18 @@
             <!--<Setter Property="Foreground" Value="{DynamicResource {x:Static vsshell:VsBrushes.CaptionTextKey}}"/>-->
             <Setter Property="FontStyle" Value="Italic"/>
         </Style>
+        <Style x:Key="Title_Paragraph" TargetType="Paragraph">
+            <Setter Property="TextAlignment" Value="Left" />
+            <Setter Property="FontFamily" Value="{DynamicResource {x:Static vsshell:VsFonts.CaptionFontFamilyKey}}"/>
+            <Setter Property="FontSize" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment155PercentFontSizeKey}}"/>
+            <Setter Property="FontWeight" Value="{DynamicResource {x:Static vsshell:VsFonts.Environment155PercentFontWeightKey}}"/>
+            <Setter Property="FontWeight" Value="SemiBold"/>
+            <Setter Property="Margin" Value="0,10,0,5" />
+        </Style>
+        <Style x:Key="SubtitleElement_Span" TargetType="Span">
```

Comment:
The generated XAML now contains a couple of new sections for the title and subtitles, so I added a style for each.